playbooks/adhoc/upgrades/upgrade.yml

```diff
@@ -250,7 +250,7 @@
     openshift_master_ha: "{{ groups['masters'] | length > 1 }}"
   tasks:
   - name: Start pcsd
-    service: name=pcsd state=started
+    service: name=pcsd enabled=yes state=started
```

Comment:
I don't see an issue with enforcing enabled here, since we are just checking/enforcing started.